### ISSUE TITLE
source-map resource urls coming out strange

### ISSUE THREAD

**Issue by DBosley** [webpack-contrib/css-loader#529]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
Using postcss-loader (only using autoprefixer plugin, no extra configurations) combined with sass-loader, css-loader, and style-loader causes the resource URL for sourcemaps to be strange:

example: http://imgur.com/a/R73hV
scss loader config
```
{
          test: /\.scss$/,
          use: [{
              loader: 'style-loader',
              options: {
                sourceMap: true,
              }
            },
            {
              loader: 'css-loader',
              options: {
                sourceMap: true,
              }
            },
            {
              loader: 'postcss-loader',
              options: {
                sourceMap: true,
                //parser: require('postcss-scss'),
              }
            },
            {
              loader: 'sass-loader',
              options: {
                includePaths: [
                  './src/styles/bootstrap',
                  './src/styles/partials'
                ],
                sourceMap: true,
              }
            },
          ],
          include: [helpers.root('src', 'styles')]
        }
```

**What is the expected behavior?**
relative urls for sourcemaps, or at least only one absolute url

**Please mention other relevant information such as your webpack version, Node.js version and Operating System.**
webpack: 2.5.1
node: 6.10.3
windows 10

**Comment by DBosley** [comment]
This issue seems to affect multiple loaders. Posted the same issue on all affected loaders:
https://github.com/webpack-contrib/css-loader/issues/529
https://github.com/webpack-contrib/sass-loader/issues/451
https://github.com/webpack-contrib/style-loader/issues/231
https://github.com/postcss/postcss-loader/issues/224

**Comment by michael-ciniawsky** [comment]
It's likely to be in `postcss-loader` :)

**Comment by DBosley** [comment]
I tried quite a few different configs today trying to figure this out. Some for scss files, some for css. For css files, I wasn't ever able to get relative paths in chrome, but by ordering my css config like this:
```
{
  test: /\.css$/,
  use: [{
      loader: 'style-loader',
    },
    {
      loader: 'css-loader',
      options: {
        sourceMap: true
      }
    },
    {
      loader: 'postcss-loader',
    }
  ],
  include: [helpers.root('src', 'styles')]
},
```
I was able to get rid of the extra absolute paths.

**Comment by michael-ciniawsky** [comment]
Should be fixed by #532, released in `v0.28.2`
